projects/igniteui-angular/src/lib/grids/grid/grid.component.spec.ts

```diff
@@ -3711,6 +3711,33 @@ describe('IgxGrid Component Tests', () => {
                     expect(overlayContent.style.display).toEqual('');
                 }));
         });
+
+        describe('Selector should display random numbers from perpage', () => {
```

Comment:
This test should be added in the grid.pagination.spec.ts. as TestBed can be used for example PagingComponent. 
There is no need test to be async. 
Tests for changing grid.perPage to default values should be added.